### DISCUSSION TITLE
local (offline) clone of an online wiki.js

### DISCUSSION THREAD

**Discussion by helgeaasen** [requarks/wiki#5552] [Help / Questions]
We are small software development team that would like to use a wiki to build our knowledge base. However, sometimes some of us can not be online but it would be great if they would still have the knowledge available that is in the wiki. 
Thus I was wondering if the following is possible:

- run a main wiki.js running "on the internet"
- clone the content and use it in an local / offline version in case no internet access is available. 

When I came across the the wiki.js feature of storage on git (https://docs.requarks.io/storage/git) I was wondering if this would make it possible? E.g. run a local wiki.js instance in a container on the local machine that interfaces with a local version of the storage, while this storage is synced via git with the online version of the wiki.

Has anyone of you done something similar or do you have other nice ideas for this situation?

**Comment by lecoredump** [comment]
That's basically the reason we chose this wiki, to be able to have an "offlineable" version. Although most people in the team simply use their preferred markdown editor to view / edit it when offline, instead of a full WikiJS container running, as it still allows doing most of what one would do through the webui directly in the Git repo (I use Obsidian for this, for example).

One thing I noted though (was coming here to open / check for an existing discussion here), is that regarding authorship, if things are added through Git, the author seems to default to the Administrator account instead of the author provided in the commit (even using the same email address). Besides that, we ran in a couple of issues with the Git storage, where some assets ended up staged but never committed, and prevented sync.

Otherwise, as long as the user knew how to use Git, we didn't note any issue that would prevent such a use.
